### PR TITLE
📄 Docs: automatically generate README.md for typescript

### PR DESCRIPTION
**Pull request by bdeneux**
## 📄 Description 

Add documentation on each typescripts package by generating each one based on the schema documentation.

## 🚀 Publication 

Each time a commit is done on contract repository, schema and README documentation is updated on this repository through a `README.md.template` located at the `ts/` root path folder by using [gomplate](https://docs.gomplate.ca/usage/) to complete fields in template.

It's avoiding rewrite for each package the installation guide and add type documentation on `README.md`.

### CODE REVIEW

File: .github/workflows/update-schema.yml [hunk 1]
```diff
@@ -74,11 +74,36 @@ jobs:
           git_user_signingkey: true
           git_commit_gpgsign: true
 
-      - name: Copy updated schema
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Install Gomplate
+        run: brew install gomplate
```
Comment (ccamel) [inline .github/workflows/update-schema.yml:81]: I'm not a fan of not fixing the versions, it poses problems of reproducibility. What I can suggest is that to reuse the recipe from the repo [okp4/networks](https://github.com/okp4/networks/blob/70ae602950ebf4bafaca7fb37681f3d3944573bb/.github/workflows/update-chains-description.yml#L58).

```suggestion
      - name: Install gomplate
        run: |
          curl -s -L -o /usr/local/bin/gomplate https://github.com/hairyhenderson/gomplate/releases/download/v3.11.5/gomplate_linux-amd64
          chmod +x /usr/local/bin/gomplate
```

**Review by ccamel — CHANGES_REQUESTED** [review]
That's great. Just one comment.

File: .github/workflows/update-schema.yml (hunk 1 above)
Comment (bdeneux) [reply .github/workflows/update-schema.yml:81]: Yes absolutely !

**Review by ccamel — APPROVED** [review]
That's brilliant! Thanks :heart:

**Review by antho31 — APPROVED** [review]
Thanks!

**Review by amimart — APPROVED** [review]
Excellent! :)

**Pull request merged**